_posts/2018-12-06-How-to-do-quick-revie-towards-tickets-and-bugs-before-release.markdown

```diff
@@ -0,0 +1,47 @@
+---
+layout: post
+title:  "How to do quick-review of tickets and bugs before release?"
+date:   2018-12-06 22:49:39 +0800
+
+---
+As we all know, before GA, there are tickets or bugs that we need to decide if they should be fixed or lower their priority or translate to Improvement.
```

Comment:
Don't do ... or ... or ...
should be fixed, have their priority lowered, or be converted to Improvement.

Reply:
Lachlan would said "As we all know" is not a proper way to convince the audience.

Reply:
It depends on the audience. If you are targeting a small group of people that you know well, then it is probably OK. 
However, we should write these blogs as if we are writing to public, so "as we all know" is best avoided.